evap/evaluation/models.py

```diff
@@ -676,19 +692,13 @@ def update_evaluations(cls):
 @receiver(post_transition, sender=Evaluation)
 def warmup_cache_on_publish(instance, target, **_kwargs):
```

Comment:
I like the way this is handled now, but `warmup_cache_on_publish` and `delete_cache_on_unpublish` do not actually do what you would think from their names now. Also, comments indicating where the actual cache generation / invalidation happens might be good.